web/src/components/Education/Education.module.css

```diff
@@ -1,72 +1,91 @@
 @import "../../colors.css";
 @import "../../fonts.css";
+
 .container {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  max-width: 90%;
+  margin: 2rem auto;
+  padding: 2rem;
+  border: 0.094rem solid var(--color-paleskyblue);
+  border-radius: 0.5rem;
   background-color: var(--color-white);
-  padding: 3rem 1.5rem;
-  max-width: 50rem;
-  margin: 0 auto;
 }
+
 h1 {
   margin-bottom: 1.875rem;
   text-align: start;
-  font-weight: 500;
-  font-size: var(--font-size-xlarge);
+  font-weight: var(--font-weight-light);
+  font-size: var(--font-size-large);
 }
 
-p {
-  font-size: var(--font-size-large);
-  color: var(--color-black);
-  margin-bottom: 1.5rem;
+h2 {
+  margin-bottom: 2rem;
+  font-size: var(--font-size-xlarge);
```

Comment:
the font size of this `h2` element is still bigger than the `h1` as mentioned before this is a bit confusing as the `h1` should be the most prominient header on the page